### PR TITLE
chore: add docs and migration guide for maps

### PR DESCRIPTION
**Pull request by paul-vd**

### CODE REVIEW

File: source/docs/advanced/features/display-a-map.md [hunk 1]
```diff
@@ -5,11 +5,38 @@ title: Display a map
...
+
+
+### Component Props  <!-- omit in toc -->
+
+We have harmonized the props of the map components to make sure that they are consistent across different components.
+#### Map Component
+
+| Property        | Description                              | Type                   |
+|-----------------|------------------------------------------|------------------------|
+| locations       | A list of locations used for the markers | [`LocationInputShape[]`](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/web/theme/components/organisms/Map/location.js#L22-31) |
```
Comment (paul-vd) [inline source/docs/advanced/features/display-a-map.md:23]: **REMINDER:** update the link lines when [MR914](https://gitlab.com/front-commerce/front-commerce/-/merge_requests/914) has been merged

File: source/docs/advanced/features/display-a-map.md [hunk 2]
```diff
@@ -5,11 +5,38 @@ title: Display a map
...
+| getMarker       | The marker node for locations            | `(location) => ReactNode` |
+| zoom            | Default zoom level                       | `number`               |
+| defaultBounds   | The default map bounds                   | [`CoordinatesShape[]`](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/web/theme/components/organisms/Map/location.js#L17-20)   |
```
Comment (paul-vd) [inline source/docs/advanced/features/display-a-map.md:26]: **REMINDER:** update the link lines when [MR914](https://gitlab.com/front-commerce/front-commerce/-/merge_requests/914) has been merged

File: source/docs/advanced/features/display-a-map.md [hunk 3]
```diff
@@ -5,11 +5,38 @@ title: Display a map
...
+| defaultCenter   | The default center for the map.          | [`CoordinatesShape` ](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/web/theme/components/organisms/Map/location.js#L17-20)    |
```
Comment (paul-vd) [inline source/docs/advanced/features/display-a-map.md:27]: **REMINDER:** update the link lines when [MR914](https://gitlab.com/front-commerce/front-commerce/-/merge_requests/914) has been merged

File: source/docs/advanced/features/display-a-map.md [hunk 4]
```diff
@@ -5,11 +5,38 @@ title: Display a map
...
+| onBoundsChanged | Event handler for bound changes          | `(event) => void`      |
+
+
+#### Marker Component
+
+| Property      | Description                            | Type                 |
+|---------------|----------------------------------------|----------------------|
+| location      | Location for the marker                | [`LocationInputShape` ](https://gitlab.com/front-commerce/front-commerce/-/blob/main/src/web/theme/components/organisms/Map/location.js#L22-31)|
```
Comment (paul-vd) [inline source/docs/advanced/features/display-a-map.md:35]: **REMINDER:** update the link lines when [MR914](https://gitlab.com/front-commerce/front-commerce/-/merge_requests/914) has been merged

**Review by abailleulfc — APPROVED** [review]

**Pull request merged**